lib/util/git/index.ts

```diff
@@ -504,13 +504,19 @@ export function getBranchCommit(branchName: string): CommitSha | null {
 }
 
 export async function getCommitMessages(): Promise<string[]> {
```

Comment:
we should move this to the `ScmPlatform` in separate PR, so it's easier to override.